composer.json

```diff
@@ -26,7 +26,8 @@
         "illuminate/filesystem": "5.0.*",
         "phpunit/phpunit": "~4.0",
         "mockery/mockery": "~0.9",
-        "squizlabs/php_codesniffer": "~2.0"
+        "squizlabs/php_codesniffer": "~2.0",
+        "fabpot/php-cs-fixer": "1.9.*"
```

Comment:
Why not just use https://styleci.io/ and ditch the cs checks from the tests. The tests and the cs should be separate. StyleCI will still use github's status API on pull requests.